### ISSUE TITLE
Something misleading about this output...

### ISSUE THREAD

**Issue by nigels-com** [Nicoretti/prysk#143]
Came across this odd situation today, noting it here for chasing another time.

This works fine: (seems legit that + is greedy)
```
  $ printf "[2022-11-16 17:57:20.398] [console] [info] Hello world v1.0.5+56.a7eb5dea\n"
  \[.+\] Hello world v.* (re)
```

However, this fails: (with extra space before Hello)
```
  $ printf "[2022-11-16 17:57:20.398] [console] [info] Hello world v1.0.5+56.a7eb5dea\n"
  \[.+\]  Hello world v.* (re)
```

Somewhere the [console] and [info] parts got eaten somehow?
(The three spaces besides the three [] parts are still there)

```
!
--- test/integration/prysk/regex.t
+++ test/integration/prysk/regex.t.err
@@ -30,4 +30,4 @@
   [.+\] Hello world v.* (re)
 
   $ printf "[2022-11-16 17:57:20.398]   Hello world v1.0.5+56.a7eb5dea\n"
-  [.+\]  Hello world v.* (re)
+  [2022-11-16 17:57:20.398]   Hello world v1.0.5+56.a7eb5dea
```

**Comment by Nicoretti** [comment]
@nigels-com thanks for reporting

**Comment by Nicoretti** [comment]
Hi @nigels-com,

the missing `[console] [info]` in the prysk output of the called command defiently seems to be odd.

Regarding the mismatch when you add an extra space, that is to be expect.
You are trying to get a full match right? 

Having something like this `\[.+\]\s*Hello world v.*`  should deal with the fact that there may be arbitrary whitespaces after the last `]`.

With grouping it may be a bit more obvious:
![image](https://user-images.githubusercontent.com/600911/204600030-cc7e68a8-5bfb-4c1b-91b2-52f9734cbe93.png)

**Comment by nigels-com** [comment]
Yes it was the diff output that I thought was possibly some sort of prysk bug.
```
+  [2022-11-16 17:57:20.398]   Hello world v1.0.5+56.a7eb5dea
```
